deployment/RS01SBPS_Deploy.csv

```diff
@@ -207,23 +207,23 @@ AT50-29,,,,RS01SBPS-SF01A-3D-SPKIRA101,12,1,2024-08-09T20:17:00,2025-08-05T14:30
-AT50-41,,,,RS01SBPS-SF01A-4F-PCO2WA101,13,1,2025-08-30T00:00:00,,ATAPL-67762-20001,ATAPL-68870-00140,ATAPL-58336-00003,44.515161,-125.389899,,N/A,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
+AT50-41,,,,RS01SBPS-PC01A-05-ADCPTD102,12,1,2025-08-12T15:45:00,,ATAPL-67762-20001,ATAPL-69839-00104,ATAPL-58315-00004,44.515161,-125.389899,,195,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
+AT50-41,,,,RS01SBPS-PC01A-06-VADCPB101,2,1,2025-08-12T15:45:00,,ATAPL-67762-20001,ATAPL-69839-00104,ATAPL-78520-00001,44.515161,-125.389899,,195,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
+AT50-41,,,,RS01SBPS-PC01A-07-CAMDSC102,12,1,2025-08-12T15:45:00,,ATAPL-67762-20001,ATAPL-69839-00104,ATAPL-78437-00011,44.515161,-125.389899,,195,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
+AT50-41,,,,RS01SBPS-PC01A-08-HYDBBA103,12,1,2025-08-12T15:45:00,,ATAPL-67762-20001,ATAPL-69839-00104,ATAPL-58324-00007,44.515161,-125.389899,,195,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
+AT50-41,,,,RS01SBPS-PC01A-4A-CTDPFA103,12,1,2025-08-12T15:45:00,,ATAPL-67762-20001,ATAPL-69839-00104,ATAPL-66662-00009,44.515161,-125.389899,,195,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
+AT50-41,,,,RS01SBPS-PC01A-4A-DOSTAD103,12,1,2025-08-12T15:45:00,,ATAPL-67762-20001,ATAPL-69839-00104,ATAPL-58320-00011,44.515161,-125.389899,,195,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
+AT50-41,,,,RS01SBPS-PC01A-4B-PHSENA102,12,1,2025-08-12T15:45:00,,ATAPL-67762-20001,ATAPL-69839-00104,ATAPL-58337-00003,44.515161,-125.389899,,195,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
+AT50-41,,,,RS01SBPS-PC01A-4C-FLORDD103,12,1,2025-08-12T15:45:00,,ATAPL-67762-20001,ATAPL-69839-00104,ATAPL-58322-00002,44.515161,-125.389899,,195,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
+AT50-41,,,,RS01SBPS-SF01A-2A-CTDPFA102,13,1,2025-08-12T15:45:00,2025-08-13T09:00:00,ATAPL-67762-20001,ATAPL-68870-00140,ATAPL-66662-00008,44.515161,-125.389899,,N/A,2904,The following parameters are preliminary and will be verified and updated after deployment: mooringID; lat; lon; deployment_depth; water_depth.,,
```

Comment:
I think you used the local times, and also, did you end the wrong deployment? I think 12 should have end dates and 13 should be open-ended?